src/app/client/src/app/modules/dashboard/components/report/report.component.ts

```diff
@@ -607,8 +608,11 @@ export class ReportComponent implements OnInit {
   }
 
   selectedTabChange(event) {
+
     const { type, downloadURL } = _.get(event, 'tab.textLabel');
     this.showExportsOption = ['chart', 'download'].includes(type);
+    this.showResetFilter = ['chart'].includes(type);
```

Comment:
@rakeshSgr Reset button not required for tables?

Reply:
What if the table have filters and config ?

Reply:
@yravinderkumar33 Global reset filter is not required here, We have reset filters in sb-table.component separately